ironfish-cli/src/commands/wallet/multisig/dkg/create.ts

```diff
@@ -2,102 +2,228 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-import { RpcClient } from '@ironfish/sdk'
+import {
+  deserializePublicPackage,
+  deserializeRound2CombinedPublicPackage,
+} from '@ironfish/rust-nodejs'
+import { AccountFormat, Assert, encodeAccountImport, RpcClient } from '@ironfish/sdk'
+import { Flags } from '@oclif/core'
 import { IronfishCommand } from '../../../../command'
 import { RemoteFlags } from '../../../../flags'
 import * as ui from '../../../../ui'
+import { Ledger } from '../../../../utils/ledger'
 
 export class DkgCreateCommand extends IronfishCommand {
   static description = 'Interactive command to create a multisignature account using DKG'
 
   static flags = {
     ...RemoteFlags,
+    participant: Flags.string({
+      char: 'n',
+      description: 'The name of the secret to use for encryption during DKG',
+    }),
+    newAccount: Flags.string({
```

Comment:
I think that handling the account name separately from the participant name opens us up to a lot of difficult edge cases if/when the names don't match

We might need to add RPCs for renaming and/or deleting identities from the wallet db in case users get 'stuck' with an identity name they don't want, but I think we should do whatever we can to try to make the names match